elastic/integrations
# Integration ti_eclecticiq

# Enhancement

## Proposed commit message

Add Eclectic IQ integration with Advanced CSV Observables Outgoing Feed data stream

## Checklist


## Author's Checklist

- [ ] Review documentation
- [ ] Review manifest files

## How to test this PR locally

Prerequiseites
- Advanced CSV Observables Outgoing Feed with populated dataset(s)
- EIQ IC API key
Steps:
- Install the integration
- Set up the integration
- Check "logs-ti_eclecticiq_latest.observables-*" index for data. Give it some time to populate, should be a couple of minutes.

## Related issues

- Relates [DEV-5635](https://eclecticiq.atlassian.net/browse/DEV-5635)


## Screenshots

We'll do this for the MR to the public repo.
